lib/sequent/rake/tasks.rb

```diff
@@ -0,0 +1,104 @@
+require 'active_record'
+require 'rake'
+require 'rake/tasklib'
+
+require 'sequent/support'
+
+module Sequent
+  module Rake
+    class Tasks < ::Rake::TaskLib
+      include ::Rake::DSL
+
+      DEFAULT_OPTIONS = {
+        migrations_path: 'db/migrate',
+        event_store_schema: 'public'
+      }
+
+      attr_reader :options
+
+      def initialize(options)
+        @options = DEFAULT_OPTIONS.merge(options)
+      end
+
+      def register!
+        register_db_tasks!
+        register_view_schema_tasks!
+      end
+
+      def register_db_tasks!
+        namespace :db do
+          desc 'Create the database'
+          task :create do
+            current_environments.each do |env|
+              env_db = db_config(env)
+              puts "Create database #{env_db['database']}"
+              Sequent::Support::Database.create!(env_db)
+            end
+          end
+
+          desc 'Drop the database'
+          task :drop do
+            current_environments.each do |env|
+              env_db = db_config(env)
+              puts "Drop database #{env_db['database']}"
+              Sequent::Support::Database.drop!(env_db)
+            end
+          end
+
+          task :establish_connection do
+            env_db = db_config(options.fetch(:environment))
+            Sequent::Support::Database.establish_connections(env_db, view_projection.schema_name)
+          end
+
+          desc 'Migrate the database'
+          task migrate: :establish_connection do
```

Comment:
These migrations should migrate the event store schema, which should be changed through migrations.

Although we don't prevent anyone from migrating the view, migrations should only target the event store.